scene/2d/joint_2d.cpp

```diff
@@ -289,11 +295,98 @@ real_t PinJoint2D::get_softness() const {
 	return softness;
 }
 
+void PinJoint2D::set_angular_limit_lower(real_t p_angular_limit_lower) {
+	if (angular_limit_lower == p_angular_limit_lower) {
+		return;
+	}
+	angular_limit_lower = p_angular_limit_lower;
+	queue_redraw();
+	if (is_configured()) {
+		PhysicsServer2D::get_singleton()->pin_joint_set_param(get_joint(), PhysicsServer2D::PIN_JOINT_LIMIT_LOWER, p_angular_limit_lower);
+	}
+}
+
+real_t PinJoint2D::get_angular_limit_lower() const {
+	return angular_limit_lower;
+}
+
+void PinJoint2D::set_angular_limit_upper(real_t p_angular_limit_upper) {
+	if (angular_limit_upper == p_angular_limit_upper) {
+		return;
+	}
+	angular_limit_upper = p_angular_limit_upper;
+	queue_redraw();
+	if (is_configured()) {
+		PhysicsServer2D::get_singleton()->pin_joint_set_param(get_joint(), PhysicsServer2D::PIN_JOINT_LIMIT_UPPER, p_angular_limit_upper);
+	}
+}
+
+real_t PinJoint2D::get_angular_limit_upper() const {
+	return angular_limit_upper;
+}
+
+void PinJoint2D::set_motor_target_velocity(real_t p_motor_target_velocity) {
+	if (motor_target_velocity == p_motor_target_velocity) {
+		return;
+	}
+	motor_target_velocity = p_motor_target_velocity;
+	queue_redraw();
+	if (is_configured()) {
+		PhysicsServer2D::get_singleton()->pin_joint_set_param(get_joint(), PhysicsServer2D::PIN_JOINT_MOTOR_TARGET_VELOCITY, motor_target_velocity);
+	}
+}
+
+real_t PinJoint2D::get_motor_target_velocity() const {
+	return motor_target_velocity;
+}
+
+void PinJoint2D::set_enable_motor(bool p_enable_motor) {
+	if (enable_motor == p_enable_motor) {
+		return;
+	}
+	enable_motor = p_enable_motor;
+	queue_redraw();
+	if (is_configured()) {
+		PhysicsServer2D::get_singleton()->pin_joint_set_flag(get_joint(), PhysicsServer2D::PIN_JOINT_FLAG_ENABLE_MOTOR, enable_motor);
+	}
+}
+bool PinJoint2D::get_enable_motor() const {
```

Comment:
```suggestion

bool PinJoint2D::get_enable_motor() const {
```
Leave space between functions